core/src/main/java/hudson/model/Node.java

```diff
@@ -115,6 +118,18 @@ public boolean isHoldOffLaunchUntilSave() {
     }
 
     /**
+     * {@inheritDoc}
+     * @since 1.635
```

Comment:
Careful—can be easily forgotten if merging does not happen as quickly as you had hoped. Prefer to use `TODO` or `FIXME`. @kohsuke [uses `1.THU`](https://github.com/jenkinsci/jenkins/blob/11c81cacf5776415cd68ee26f32e5d28549a0322/core/src/main/java/jenkins/security/s2m/AdminCallableMonitor.java#L19) and do not ask me [what it means](http://www.acronymfinder.com/THU.html). :-)